src/components/Nav/NavList.js

```diff
@@ -0,0 +1,8 @@
+export const NavList = [
```

Comment:
```suggestion
export const navList = [
```
컴포넌트가 아니네요. 변수명 수정 해 주세요~

Reply:
넵 수정하겠습니다 감사합니다!